test/extended/controller_manager/pull_secret.go

```diff
@@ -60,7 +60,7 @@ var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
 	defer g.GinkgoRecover()
 	oc := exutil.NewCLI("pull-secrets")
 
-	g.It("TestAutomaticCreationOfPullSecrets", func() {
+	g.It("TestAutomaticCreationOfPullSecrets [apigroup:config.openshift.io]", func() {
```

Comment:
gate on image.openshift.io please.  It may not be directly used, but when we add that API, we'll want this test.